homework8_validate.py

```diff
@@ -0,0 +1,29 @@
+"""Module providing a function printing python version."""
+number_of_credit_card = input("Enter your card number: ")
+while not number_of_credit_card.isdigit():
+    print("Errors. Number of credit card must consist of digits. ")
+    number_of_credit_card = input("Enter your correct card number: ")
```

Comment:
the same code line#2 and #5

Reply:
This line had the same variable but different meaning: 
input("Enter your card number: ")        and 
input("Enter your correct card number: ").
This is can be possible, isn't it? 
But I changed it, as you had said